simulations/vip-342/bscmainnet.ts

```diff
@@ -0,0 +1,56 @@
+import { expect } from "chai";
+import { Contract } from "ethers";
+import { ethers } from "hardhat";
+import { expectEvents } from "src/utils";
+import { forking, testVip } from "src/vip-framework";
+import { checkInterestRate } from "src/vip-framework/checks/interestRateModel";
+
+import vip342, { NEW_IR, OLD_IR, VBNB } from "../../vips/vip-342/bscmainnet";
+import VTOKEN_CORE_POOL_ABI from "./abi/VTokenCorePool.json";
+
+forking(40605412, async () => {
+  let vbnb: Contract;
+  const provider = ethers.provider;
+
+  before(async () => {
+    vbnb = new ethers.Contract(VBNB, VTOKEN_CORE_POOL_ABI, provider);
+  });
```

Comment:
I guess we can just do

```suggestion
  const provider = ethers.provider;
  const vbnb =  new ethers.Contract(VBNB, VTOKEN_CORE_POOL_ABI, provider);
```